xmppserver/src/main/java/org/jivesoftware/openfire/spi/ConnectionListener.java

```diff
@@ -1026,4 +1027,31 @@ public String toString()
                 '}';
     }
 
+    public void setStrictCertificateValidation(boolean strictCertificateValidation) {
```

Comment:
Please add Javadoc and \*twitch\* move it above the `toString()` method.

Reply:
done